src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj

```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <_ExpectedSharedFrameworkBinaries Include="@(AspNetCoreAppReference);@(AspNetCoreAppReferenceAndPackage);@(ExternalAspNetCoreAppReference);@(_TransitiveExternalAspNetCoreAppReference)" />
-    <_ExpectedSharedFrameworkBinaries Condition="'$(TargetOsName)' == 'win' AND '$(TargetArchitecture)' != 'arm'" Include="aspnetcorev2_inprocess" />
+    <_ExpectedSharedFrameworkBinaries Condition="'$(IsServicingBuild)' == 'true' OR ('$(TargetOsName)' == 'win' AND '$(TargetArchitecture)' != 'arm')" Include="aspnetcorev2_inprocess" />
```

Comment:
We need to include aspnetcorev2_inprocess in servicing build since we now use the file eng/PlatformManifest.txt instead of calculating what was built on linux and macOS.

Reply:
Probably need a special case in any comparisons against the checked-in manifest when running on Linux, macOS, or Windows ARM. With the updated condition, will try to bring in a non-existent `aspnetcorev2_inprocess` binary on those platforms.

Reply:
That is, change the test instead to ignore one line of the manifest in the non-Windows non-ARM cases